src/Microsoft.DotNet.Build.Tasks/Targets/test-runtime/project.json

```diff
@@ -36,9 +37,9 @@
        "System.Threading.Thread": "4.0.0-beta-*",
        "System.Xml.ReaderWriter": "4.0.10",
        "System.Xml.XDocument": "4.0.10",
-       "xunit": "2.1.0-beta3-*",
```

Comment:
I do not; I haven't been using VS for package restoration/building, though that issue does sound familiar. Has that behavior arisen elsewhere that I can look at?